engine/src/test/java/org/hibernate/validator/test/internal/xml/XmlMappingParserTest.java

```diff
@@ -50,21 +51,23 @@ public void testAdditionalConstraintValidatorsGetAddedAndAreLastInList() {
 				DecimalMin.class
 		);
 
-		assertEquals( validators.size(), 2, "Wrong number of default validators" );
+		assertEquals( validators.size(), 3, "Wrong number of default validators" );
```

Comment:
I feel like this test could be done in a less fragile way, but I didn't want to rewrite it without consent. My idea would be to just check whether the list is non-empty and the custom validator is the last, without also verifying the default validators explicitly.